### ISSUE TITLE
Regarding optimizer and parameters for learning in attention_ocr

### ISSUE THREAD

**Issue by rohitsaluja22** [tensorflow/models#4611]
Hey @alexgorban  and @bitfort  and others, 

This is regarding optimizer and parameters for learning in attention_ocr.
In the paper, you mention stochastic gradient descent optimizer to work and following parameters:-

--label_smoothing=0.9 --momentum=0.75 -- weight  decay=0.00004 --lstm_state_clip_value=10 --learning_rate=0.002 --learning_rate_decay=0.1 --decay_after=1,200,000 

Whereas in implementation(in train.py) you use tf.train.MomentumOptimizer and following parameters:-

--label_smoothing=0.1 --momentum=0.9 -- weight  decay=0.00004 --lstm_state_clip_value=10  --learning_rate=0.004  

Is the later better than former(which I think so)? Or shall I change to former to replicate results or try it on my data?

Yours
Rohit

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by rohitsaluja22** [comment]
Have I written custom code: N/A
OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: N/A
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
